frontend/src/components/OrganizationSearch.vue

```diff
@@ -0,0 +1,166 @@
+<!--suppress XmlInvalidId -->
+<template>
+  <div>
+    <!-- Basic Search -->
+    <v-row no-gutters>
+      <v-col class="col-6">
+        <label for="organization-search" >
+          Search
+          <v-tooltip right>
+            <template v-slot:activator="{ on }">
+              <v-icon v-on="on" small>mdi-help-circle</v-icon>
+            </template>
+            <span>Search by organization name or ID</span>
+          </v-tooltip>
+        </label>
+
+        <v-text-field
+          id="organization-search"
+          outlined
+          dense
+          v-model="organizationSearchInput"
+          placeholder="Organization or ID"    
+          @keyup.enter="searchOrganization('&search='+organizationSearchInput.replaceAll('\\','%5C'))"     
+        />
+      </v-col>
+      <v-col class="col-4">
+          <v-btn id="search-button" class="primary" medium @click.native="searchOrganization('&search='+organizationSearchInput.replaceAll('\\','%5C'))">Search Organizations</v-btn>
```

Comment:
As the "search" input field is just a client-side filter, you can remove this search button. Load all of the organizations on tab load.

Reply:
done!